spec/mongo/database_spec.rb

```diff
@@ -119,7 +119,7 @@
 
     context 'when specifying a batch size' do
 
-      it 'returns the stripped names of the collections', unless: need_to_skip_on_sharded_auth_40? do
+      it 'returns the stripped names of the collections' do
```

Comment:
Do we no longer need to skip this test (and the one below) with a sharded 4.0 cluster and auth?

Reply:
These tests do pass currently and I was unable to determine why this skip is needed.